doc/tutorial/cora-gcn.md

```diff
@@ -0,0 +1,168 @@
+# Classify Cora Dataset Using GCN
+
+This tutorial shows how to train [GCN](https://arxiv.org/pdf/1609.02907.pdf) model on the Cora dataset. In this tutorial, you will learn how to:
+- Train a GCN on the [Cora citation dataset](https://linqs-data.soe.ucsc.edu/public/lbc/cora.tgz).
+- Use the trained GCN to predict the label for some of the papers in the dataset.
+
+
+## The Dataset
+
+The Cora dataset is graph dataset about a citation network of scientific papers. It consists of 2708 scientific publications classified into one of seven classes. The citation network consists of 5429 links. Each publication in the dataset is described by a 0/1-valued word vector indicating the absence/presence of the corresponding word from the dictionary. The dictionary consists of 1433 unique words.
+
+Due to the fact that graph data is not applicable to be divided into batches, we are not able to split the data and store in training dataset and test dataset respectively. Thus, we represent the entire graph through two tables: `Node Table` and `Edge Table`. (Anyone who wants to store graph data in the database can refer to this method.)
+
+Here are the column description of the `Node Table`:
+
+Column | Explain 
+-- | -- 
+id| Id for the node.
+name| Name for the node.
+features| Feature vector of the node represented in the string. An example would be "0,0,1".
+label | Label for the node.
+
+The following command can be used to construct the `Node Table`. 
+
+```sql
+CREATE TABLE cora.node (
+        id INT,
+        node_name TEXT,
+        features  TEXT,
+        label TEXT);
+```
+
+Here are the column description of the `Edge Table`:
+
+Column | Explain 
+-- | -- 
+id| Id for the edge.
+from_node_id| Id for the from node of the edge.
+to_node_id| Id for the to node of the edge.
+weight | Weight for the edge.
+
+The following command can be used to construct the `Edge Table`. 
+
+
+```sql
+CREATE TABLE cora.edge (
+        id INT,
+        from_node_id INT,
+        to_node_id  INT,
+        weight FLOAT);
+```
+
+You can have a quick peek of the data by running the following standard SQL statements.
+
+```sql
+%%sqlflow
+describe cora.node;
+describe cora.edge;
+```
+
+```sql
+%%sqlflow
+select * from cora.node limit 10;
+```
+
+## Train GCN on the Cora dataset
+
+Let's train a GCN model!
+
+### Load data from Cora
+You can load the data from database following a standard SQL command such as `SELECT * FROM cora.node`. However, since GCN is supposed to deal with graph data, you have to load both the `Node Table` and `Edge Table` at once.
```

Comment:
You can load the data from **the** database following a standard SQL command such as `SELECT * FROM cora.node`. However, since **the** GCN **model** is supposed to deal with **the** graph data, you have to load both the `Node Table` and `Edge Table` at once.